Improve taggings-modifier's style in service catalogs

jeffibm opened the pull request:
Before
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/10bf0922-5097-413a-a00f-f3dfa1692fc7)
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/173435db-6b2c-4c72-8e2b-348439c2f42c)

After
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/e4679847-5c08-42a9-928d-b185766dfd61)
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/9f45d20d-69ac-4eb9-9327-426bbd953fb2)

When more items are selected, the container displays a scrollbar
<img width="1128" alt="image" src="https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/08eddd7f-7100-4480-80e6-de40ee7d6b5d">

DavidResende0 reviewed (approved):
Overall LGTM, Only thing I'd suggest is that we could maybe make it so the assigned tags section is scrollable rather then letting it grow infinitely. Just something we could consider for a follow up.
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/64800041/8cd1dbbd-02fd-4337-967b-1f98ca34685a)

jeffibm commented:
> Overall LGTM, Only thing I'd suggest is that we could maybe make it so the assigned tags section is scrollable rather then letting it grow infinitely. Just something we could consider for a follow up. ![image](https://private-user-images.githubusercontent.com/64800041/294589608-8cd1dbbd-02fd-4337-967b-1f98ca34685a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ2OTg5MjAsIm5iZiI6MTcwNDY5ODYyMCwicGF0aCI6Ii82NDgwMDA0MS8yOTQ1ODk2MDgtOGNkMWRiYmQtMDJmZC00MzM3LTk2N2ItMWY5OGNhMzQ2ODVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA4VDA3MjM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjMmQwZjFhMjAzNDU5MmI5NzY0YTVmZGRkMDM0NjEzZjZhNDAzNmZjZDg2MGIzNTY0MjI5ZjIwOTBmMTZmNDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wk1nrcn-TBupnK3D_t8fIZOpPX3IZW8FnzFk5trnE_c)

Hey @DavidResende0 , I added a scrollbar as suggested. 
I have updated the screenshot in the PR description.

DavidResende0 commented:
> > Overall LGTM, Only thing I'd suggest is that we could maybe make it so the assigned tags section is scrollable rather then letting it grow infinitely. Just something we could consider for a follow up. ![image](https://private-user-images.githubusercontent.com/64800041/294589608-8cd1dbbd-02fd-4337-967b-1f98ca34685a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ2OTg5MjAsIm5iZiI6MTcwNDY5ODYyMCwicGF0aCI6Ii82NDgwMDA0MS8yOTQ1ODk2MDgtOGNkMWRiYmQtMDJmZC00MzM3LTk2N2ItMWY5OGNhMzQ2ODVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA4VDA3MjM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjMmQwZjFhMjAzNDU5MmI5NzY0YTVmZGRkMDM0NjEzZjZhNDAzNmZjZDg2MGIzNTY0MjI5ZjIwOTBmMTZmNDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wk1nrcn-TBupnK3D_t8fIZOpPX3IZW8FnzFk5trnE_c)
> 
> Hey @DavidResende0 , I added a scrollbar as suggested. I have updated the screenshot in the PR description.

Looks Great. @GilbertCherrie can you do a final review?
